test/remote/gateways/remote_authorize_net_cim_test.rb

```diff
@@ -73,11 +73,30 @@ def test_successful_profile_create_get_update_and_delete
     assert_success response
     assert_nil response.authorization
     assert response = @gateway.get_customer_profile(customer_profile_id: @customer_profile_id)
-    assert_nil response.params['profile']['merchant_customer_id']
```

Comment:
I am not sure if this is based on previous behavior, but after a get request these are always present.